README.md

```diff
@@ -33,7 +33,63 @@ app.use(require('webpack-hot-middleware')(compiler, {
 }));
 ```
 
-#### webpack-dev-middleware
+#### package.json (recommended)
+
+After the initial release, it was decided that running a separate process and communicating via sockets would be more efficient and solves a lot of problems with stdout.
+
+The new way to run the dashbaord is to add the plugin, and then call the provided binary.
+
+First, import add the plugin to your webpack config, or apply it to your compiler:
```

Comment:
Might be nice to show require path to `DashboardPlugin()` to make it easier for people to quickly setup.

ie: `const DashboardPlugin = require('webpack-dashboard/plugin');`